docs/aws-region-enable.md

```diff
@@ -27,24 +27,33 @@ This document provides the information on how to address and resolve missing AWS
     In the dialog box, review the informational text and choose Enable Region.
     Wait until the Region is ready to use.
 
-    We would also like to get AWS public access enabled for this region. Please use the following command:
+    We would like to get AWS public access enabled for this region. Please use the following command:
         "aws ec2 disable-image-block-public-access --region `<missing-region>`"
 
-    Let's also adjust the quota to match the other regions :
-        "aws service-quotas request-service-quota-increase --region=`<missing-region>` --service-code=ec2 --quota-code=L-0E3CBAB9 --desired-value 1000.00"
+    We would also like the quota level to be set, the quota codes can be viewed using the below command:
+        "aws service-quotas --service-code ec2 list-service-quotas"
```

Comment:
The command order is wrong so let's fix that. and let's include specifying the region here like we do in the below command so we ensure when they check the quota number it is > `1000` they are looking at the right region.

```suggestion
        "aws service-quotas list-service-quotas --service-code=ec2 --quota-code=L-0E3CBAB9 --region=ap-southeast-5
```

Reply:
Changed the order of the command as suggested